plan/physical_plan_test.go

```diff
@@ -596,7 +596,7 @@ func (s *testPlanSuite) TestProjectionElimination(c *C) {
 		},
 		{
 			sql: "select t1.a from t t1 where t1.a in (select t2.a from t t2 where t1.a > 1)",
-			ans: "Apply{Table(t)->Table(t)->Cache->Selection}->Selection->Projection",
+			ans: "Apply{Table(t)->Table(t)->Selection}->Selection->Projection",
```

Comment:
Currently physical inititialize and addCache is called after the ProjectionElimination, so cache doesn't appear in the TestProjectionElimination.